src/services/blogApiServices.js

```diff
@@ -0,0 +1,39 @@
+export const getUser = id => {
```

Comment:
```suggestion
const ROOT_ENDPOINT = 'https://jsonplaceholder.typicode.com';
const USERS_URL = `${ROOT_ENDPOINT}/users`;
const POSTS_URL = `${ROOT_ENDPOINT}/posts`;
const COMMENTS_URL = `${ROOT_ENDPOINT}/comments`;

export const getUser = id => {
  return fetch(`${USERS_URL}/${id}`)
    .then(res => res.json());
};

export const getUsers = () => {
  return fetch(USERS_URL)
    .then(res => res.json());
};

export const getUserPosts = userId => {
  return fetch(`${POSTS_URL}?userId=${userId}`)
    .then(res => res.json());
};

export const getPosts = () => {
  return fetch(POSTS_URL)
    .then(res => res.json());
};

export const getPost = postId => {
  return fetch(`${POSTS_URL}/${postId}`)
    .then(res => res.json());
};

export const getComments = () => {
  return fetch(COMMENTS_URL)
    .then(res => res.json());
};

export const getComment = commentId => {
  return fetch(`${COMMENTS_URL}/${commentId}`)
    .then(res => res.json());
};

export const getPostComments = postId => {
  return fetch(`${COMMENTS_URL}?postId=${postId}`)
    .then(res => res.json());
};
```
Another way to setup your api file :)